docs/getting_started_guide/Installation_Options/ISO/Building_A_SIMP_ISO.rst

```diff
@@ -8,12 +10,13 @@ SIMP ISO.
 The following guides provide an overview of the supported build methods.
 
 .. WARNING::
-  Prior to starting any build method, you will need to ensure that you follow
-  the instructions in :ref:`gsg-environment_preparation`.
+
+    Prior to starting any build method, you will need to ensure that you follow
+    the instructions in :ref:`gsg-environment_preparation`.
 
 .. toctree::
-  :maxdepth: 2
+   :maxdepth: 2
 
```

Comment:
Needs something upfront (table) that tells when you would want to build from tar, when you would want to build from source, when you need to build on a machine that has the target OS (RedHat?), when you build for either CentOS 6 or CentOS 7 from docker, and when you have to build with older selinux.  The information (if available) is scattered.  As a person who may want to build the ISO, I want a decision tree/table up front to guide me to the correct section.

Reply:
@lnemsick-simp Can you create a ticket for this? While it would be quite useful I think it's going to have to be part of a larger overhaul.

To date, most people have used the ISO and the rest have use the RPMs. Very few have tried to build everything from scratch.

Reply:
New ticket was entered: https://simp-project.atlassian.net/browse/SIMP-5382